ipatests/test_integration/test_ipahealthcheck.py

```diff
@@ -21,7 +21,7 @@
 HEALTHCHECK_LOG_ROTATE_CONF = "/etc/logrotate.d/ipahealthcheck"
 HEALTHCHECK_LOG_DIR = "/var/log/ipa/healthcheck"
 HEALTHCHECK_OUTPUT_FILE = "/tmp/output.json"
-HEALTHCHECK_PKG = ['freeipa-healthcheck']
+HEALTHCHECK_PKG = ["freeipa-healthcheck"]
```

Comment:
Can you update it  to  `HEALTHCHECK_PKG = ["*ipa-healthcheck"]` ?

Reply:
@mrizwan93 have made this change as well to ensure it works for other distros.